### ISSUE TITLE
Вычистить директорию .idea из истории Git и добавить в исключения файлы других IDE

### ISSUE THREAD

**Issue by ri-gilfanov** [amironov73/PythonIrbis#9]
В файле .gitignore директория `/.idea` указана и изменения в ней отслеживаться не должны -- всё-таки у каждого разработчика файлы в ней обычно различаются.

Однако, директория присутствует в репозитории на Github. Очевидно, она была закоммичена в репозиторий по ошибке и только потом был добавлен файл .gitignore.

Можно вычистить данную директорию из истории изменений Git, но ценой изменения hash-сумм всех коммитов. То есть, разработчикам склонировавшим себе данный репозиторий придётся клонировать его повторно, чтобы синхронизировать историю.

Так же имеет смысл добавить в исключения файлы и директории, которые могут создавать другие популярные IDE.

Если никто не против, могу заняться.

**Comment by amironov73** [comment]
Согласен. По недосмотру в коммит попали файлы, всё никак не вычищу. :)

**Comment by ri-gilfanov** [comment]
Вроде готово:

* ветка master очищена от директории .idea;
* ветка pypi очищена от директории .idea;
* в ветку master сделан commit с дополнительными правилами .gitignore для IDE и текстовых редакторов.

**Comment by amironov73** [comment]
Для информации: ветка pypi предназначена для публикации пакета на pypi.org. Когда appveyor.com видит коммит в этой ветке, активируется заливка на "боевой" репозиторий pypi.

Наверное, это нецелевое использование механизма бранчей в Git, но мне так удобно. :)

**Comment by amironov73** [comment]
Сейчас благодаря опечатке в YML-файле, публикация не произошла, что неплохо. :)

**Comment by ri-gilfanov** [comment]
Я догадался по названию ветки, поэтому ни прямых коммитов в неё, ни тем более слияния из master делать не стал.

Как я понимаю, слияние master в pypi лучше делать после покрытия тестами новой функциональности.

**Comment by amironov73** [comment]
План действий такой:

1. Покрываем тестами новый функционал.
2. Документируем его (желательны примеры, т. к. функционал нетривиальный).
3. Мерджим master в pypi.
4. Меняем версию на 0.2. Заодно исправляем опечатку в YML-файле.
5. Пушим и радуемся новой версии пакета на pypi.
